### PR TITLE
Login screen Redirect and background and implemented account register

### PR DESCRIPTION
**Pull request by Ray-R4y**
#22 #21 
Implemented the code to redirect the user to the home screen after successful login. Also added background to the login screen

### CODE REVIEW

**Comment by UntidyRAM** [comment]
It looks good. Just remember to not capitalise the words in CSS classes. You have `.Background`, it should rather be `.background`.

File: src/services/login_db.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ async function LogIn (username, password) {
       resolve('Username and password are invalid.')
     } else if (/^[a-zA-Z]+$/.test(username) === false) {
```
Comment (UntidyRAM) [inline src/services/login_db.js:18]: Shouldn't usernames also be allowed to have numbers and some special characters?

**Review by UntidyRAM — COMMENTED** [review]
It looks good, you just need to fix the capitalisation issue in the CSS file and fix the username input check. After that, you can merge 😊

File: src/services/login_db.js (hunk 1 above)
Comment (Ray-R4y) [reply src/services/login_db.js]: updated and fixed

**Pull request merged**